test/e2e/storage/testsuites/volumemode.go

```diff
@@ -196,45 +195,9 @@ func (t *volumeModeTestSuite) defineTests(driver TestDriver, pattern testpattern
 				}()
 				framework.ExpectError(err)
 			})
-		} else {
```

Comment:
VolumeMode tests now seem to be only for block volume mode tests for block unsupported driver.
So, `testpatterns.FsVolModePreprovisionedPV` and `testpatterns.FsVolModeDynamicPV` in `testPatterns` for the volumeMode tests seem to be unnecessary.
(Then, it would be better to  also remove `VolMode` check and filesystem case.)

Reply:
I'm leaving the patterns there for now, I have plans for other tests where filesystem will be useful (use FS PV as block in a Pod and expect it to fail and the same with a block PV used as FS in a Pod).

Reply:
https://github.com/kubernetes/kubernetes/pull/79796 seems useful. So, I agree to leave the patterns there for this commit.